src/Servers/Kestrel/test/InMemory.FunctionalTests/Http2/Http2TimeoutTests.cs

```diff
@@ -582,7 +583,7 @@ public async Task DATA_Sent_TooSlowlyDueToOutputFlowControlOnMultipleStreams_Abo
             await SendDataAsync(3, _maxData, endStream: true);
 
             await ExpectAsync(Http2FrameType.HEADERS,
-                withLength: 32,
+                withLength: 2,
```

Comment:
Why did the length change?

Reply:
This test was likely written before we added some HTTP2 feature that allowed this to be smaller. Other tests in this file that expect 2 HEADERS frames also expect length 2 for the second one.